doc/releases/release-notes-3.7.rst

```diff
@@ -26,6 +26,59 @@ Mbed TLS was updated to version 3.6.2 (from 3.6.0). The release notes can be fou
 Mbed TLS 3.6 is an LTS release that will be supported
 with security and bug fixes until at least March 2027.
 
+MCUboot
+*******
+
+The following commits from MCUboot have been cherry-picked into the ``v3.7-branch`` for Zephyr:
+
+  * boot: zephyr: use EXTRA_CONF_FILE instead of deprecated OVERLAY_CONFIG
+  * boot: zephyr: Fix serial recovery for NXP IMX.RT platforms
+  * boot: zephyr: Fix Warning 'boot_serial_enter defined but not used'
+  * zephyr: hello_world: Fix the main() return type warning
+  * bootutil: Fix AES and SHA-256 contexts not zeroized with mbedTLS
+  * ci: Fix zephyr workflow
+  * bootutil: Add better mode selection checks
+  * ci: Fix FIH Docker image release usage
+  * ci: Update FIH docker to Ubuntu Jammy (22.04)
+  * boot: zephyr: board: Fix nrf54l15pdk Kconfig fragment
+  * boot: zephyr: MCXN947 currently only does not support swap mode
+  * boot_serial: Fix format warning
+  * boot_serial: Fix incompatible-pointer-types warning
+  * bootutil: loader: Verify image header before checking image
+  * boot: main: avoid unused build warning
+  * bootutil: Fix missing include
+  * bootutil: Fix swap move max app size calculation
+  * imgtool: Add missing encodings to emitter tests
+  * imgtool: Fix getpub fails for ed25519 key
+  * docs: release-notes: Add note on name clash fix
+  * zephyr: Fix issue with sysbuild if something else is named mcuboot
+  * Fix style issues
+  * boot: Change boot_enc_load to take slot number instead of image
+  * docs: release-notes: Add note on fixed zephyr RAM load address
+  * boot: zephyr: Fix RAM load chain load address
+  * imgtool: Bump cryptography library version
+  * boot: Move encryption context invalidation to boot_enc_drop.
+  * boot: Rename boot_enc_decrypt to boot_decrypt_key
+  * boot: boot_serial: change logging to debug level
+  * doc: readme-zephyr: fix the scratch partition example
+  * boot: Add missing boot_enc_init
+  * boot: Remove pointless slot identification
+  * sim: Fix MCUBOOT_SWAP_USING_SCRATCH defined in direct-xip and ram-load
+  * bootutil: Properly retrieve image headers after interrupted swap-scratch
+  * docs: release-notes: Add note on fixed nordic boards
+  * boot: zephyr: Fix build for nrf9160dk
+  * boot: zephyr: Fix build for thingy53
+  * boot: zephyr: Fix build for thingy52
+  * Revert "boot: Add MCUBOOT_HW_KEY support for image encryption"
+  * boot: Fix ASN.1 for mbedtls >= 3.1
+  * ext: fiat: Use user-defined assert macro
+
+Trusted Firmware-M
+******************
+
+* TF-M was updated to version 2.1.1 (from 2.1.0).
+  The release notes can be found at: https://trustedfirmware-m.readthedocs.io/en/tf-mv2.1.1/releases/2.1.1.html
+
```

Comment:
```suggestion

TF-M 2.1 is an LTS release synchronized with Mbed TLS 3.6 that
will be supported with security and bug fixes until March 2027.
```

Reply:
Updated, also changed the lines above to match the mbedtls update at the top